tests/test_main.cpp

```diff
@@ -15,6 +16,18 @@ void init_global_game_state() {
     PATH_INFO::init_user_dir("./");
     PATH_INFO::set_standard_filenames();
 
+    if( !assure_dir_exist( FILENAMES["config_dir"] ) ) {
+        assert( !_( "Unable to make config directory. Check permissions." ) );
```

Comment:
Alright, I've nuked the translation.

In general I agree that the asserts seem evil, but you did find my line of reasoning. It's the test suite. If we *did* try to soldier on, then we'd just break during the world generation assert further down, and it would be harder to figure out *why* the assertion failure occurred. So really, exiting with an assertion failure and nice message here is probably the best case scenario.